<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui</class>
 <widget class="QMainWindow" name="gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>micamera</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="initCamera">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>init camera</string>
    </property>
   </widget>
   <widget class="QLabel" name="viewImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>360</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string>viewImage</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_photo">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>create image</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeCamera">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>close camera</string>
    </property>
   </widget>
   <widget class="QPushButton" name="transformImage">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>transform Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_text">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>68</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>treshold</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_treshold">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>160</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_continue">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>70</y>
      <width>99</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>continue</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m1l">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>motor 1 left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_m1r">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>motor 1 right</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
